app/components/channel_list/header/header.tsx

```diff
@@ -89,17 +83,9 @@ const ChannelListHeader = ({displayName, iconPad}: Props) => {
                     <CompassIcon
                         style={styles.plusIcon}
                         name={'plus'}
-                        onPress={async () => {
```

Comment:
The `onPress` before was being used to show the loading / error components (not needed to show anymore), but yes - it will eventually open up a category menu